intersection-observer/index.html

```diff
@@ -0,0 +1,40 @@
+---
+feature_name: IntersectionObserver
+chrome_version: 51
+feature_id: 5695342691483648
+---
+
+<h3>Background</h3>
+<p>
+  The <a href="https://wicg.github.io/IntersectionObserver/">IntersectionObserver API</a>
+  provides an API to understand the visibility and position of DOM elements relative to
+  a containing element or to the top-level viewport.
+</p>
+<p>
+  This sample illustrates using <code>IntersectionObserver</code> to implement a infinite
+  scroller without having to rely on scroll events. A sentinel element triggers the loading
+  of additional elements once it comes into view and is being recycled after the new
+  elements have been attached to the list.
+<p>
+  The <a href="https://developers.google.com/web/updates/???"><em>Google Devlopers Web Updates</em></a>
```

Comment:
As the article's author, you get to choose the final part of the URL's path, and the prefix is going to be https://developers.google.com/web/updates/2016/04/. You should be able to put in the actual value here, and we can hold off on merging until your article is published, if you'd like.